.github/workflows/ci.yml

```diff
@@ -93,23 +48,15 @@ jobs:
     - run: npm run generate
       env:
         CRAFT_ENDPOINT: ${{ secrets.CRAFT_ENDPOINT }}
-        LIBCAL_ENDPOINT: ${{ secrets.LIBCAL_ENDPOINT }}
```

Comment:
I think we need to keep LIBCAL_ENDPOINT? (Pointing to the proxy though - and we probably don't need to store it as a secret though that won't hurt anything)

Reply:
Okay I will add this back

Reply:
I am doing this in nuxt config 
```
/*
   * Axios. Used for Libcal query.
   */
    axios: {
        baseURL:
      process.env.LIBCAL_ENDPOINT ||
      'https://test.proxy.calendar.library.ucla.edu/',
    },
```